packages/atomic/src/components/commerce/atomic-commerce-interface/atomic-commerce-interface.new.stories.tsx

```diff
@@ -43,6 +43,7 @@ export const SearchBeforeInit: Story = {
     const commerceInterface = context.canvasElement.querySelector(
       'atomic-commerce-interface'
     );
+    await customElements.whenDefined('atomic-commerce-interface');
```

Comment:
Racing condition introduced by the new loading mechanism; component may load later.

Reply:
Aux Issue #2: Resiliency improvements